features/facebookImport/test/ministory/json-file-names-analysis.test.js

```diff
@@ -0,0 +1,86 @@
+import JSONFileNamesAnalysis from "../../src/model/analyses/report/json-file-names-analysis";
+import commonStructure from "../../src/static/commonStructure";
+import { ZipFileMock } from "@polypoly-eu/poly-import";
+import { runAnalysisForExport } from "../utils/analyses-execution";
+import { expectAnalysisSuccessStatus } from "../utils/analysis-assertions";
+
+describe("JSON files analysis for empty zip", () => {
+    let status = null;
+
+    beforeAll(async () => {
+        let zipFile = new ZipFileMock();
+        const { analysisResult } = await runAnalysisForExport(
+            JSONFileNamesAnalysis,
+            zipFile
+        );
+        ({ status } = analysisResult);
+    });
+
+    it("has success status", async () => {
+        expectAnalysisSuccessStatus(status);
+    });
+});
+
+describe("JSON files analysis for non-empty zip", () => {
+    let status = null;
+    let result = null;
+    const commonJsonFiles = commonStructure
+        .filter((path) => path.match(/\.json$/))
+        .map((jsonPath) => {
+            return jsonPath.substring(1);
+        });
+    beforeAll(async () => {
+        let zipFile = new ZipFileMock();
+        commonJsonFiles.forEach((jsonPath) => {
+            zipFile.addJsonEntry(jsonPath, { foo: "bar" });
+        });
+        const { analysisResult } = await runAnalysisForExport(
+            JSONFileNamesAnalysis,
+            zipFile
+        );
+        result = analysisResult.analysis;
+        ({ status } = analysisResult);
+    });
```

Comment:
Nit: Seems like there's a bit of duplication in the test setup that you could eliminate, IMHO, especially tests should be DRY. But not a blocker for me - tests with some duplication are way better than no tests.

Reply:
I've put all common code in a single function, and now there's no repetition... And there's a test more.